catalyst/exchange/exchange_blotter.py

```diff
@@ -171,6 +171,9 @@ def order(self, asset, amount, style, order_id=None):
             return None
 
         if self.simulate_orders:
+            if asset.exchange == 'bitfinex' and order_id is None:
+                order_id = str(np.random.binomial(1 << 40, 0.5))
+
```

Comment:
`binomial()` is to generate an `int64` ID specified by Bitfinex.

A special condition added here isn't the most elegant, but it's straightforward enough. If you know a better way to refactor, please let me know.